modules/standard/UtilReplicatedVar.chpl

```diff
@@ -115,10 +144,18 @@ proc rcCollect(replicatedVar: [?D] ?MYTYPE, collected: [?CD] MYTYPE): void
       col = replicatedVar[rcDomainIx];
 }
 
+/*
+Access the copy of `replicatedVar` on the current locale.
+
+This is equivalent to ``replicatedVar[1]``.
```

Comment:
Yup. I added a comment to my code example: "must use 1".
Good that you brought it up.